README.md

```diff
@@ -28,6 +28,8 @@ The JSON naming convention follows Hypertext Application Language (HAL).
 |[```/v1/api/products/{productId}```](docs/api_specification.md#get-v1apiproductsproductid)        | GET    |  Gets an existing product with the specified Id   |
 |[```/v1/api/products?gatewayAccountId={gatewayAccountId}```](docs/api_specification.md#get-v1apiproductsgatewayaccountid)        | GET    |  Gets lists of products that belongs to a gateway account   |
 |[```/v1/api/products/{productId}```](docs/api_specification.md#delete-v1apiproductsproductid)        | DELETE    |  Deletes/Disables the product with the specified Id   |
-|[```/v1/api/products/{productId}/items```](docs/api_specification.md#post-v1apiproductsproductiditems)        | POST    |  Creates a new instance of a product that will take a payment  |
+|[```/v1/api/payments```](docs/api_specification.md#post-v1apipayments)        | POST    | Creates a new payment                        |
+|[```/v1/api/payments/{paymentId}```](docs/api_specification.md#get-v1apipaymentspaymentid) |  GET  |     Gets an existing payment    |
+|[```/v1/api/payments?productExternalId={productId}```](docs/api_specification.md#get-v1apipaymentsproductid) | GET | Gets a list of payments that belong to a product  |
```

Comment:
To retrieve a list of payment related to a product wouldn't it be cleaner to do it this way:

```/v1/api/products/{productId}/payments```